cluster-api/src/test/java/io/aiven/klaw/clusterapi/ClusterApiControllerIT.java

```diff
@@ -425,6 +370,62 @@ public void createAclConsumerPrincipal() throws Exception {
     assertThat(aclBindings1.size()).isEqualTo(1);
   }
 
+  @Test
+  @Order(8)
+  public void createTopicsWhenTopicAlreadyExists() throws Exception {
+    String topicName = "testtopic";
+    ClusterTopicRequest clusterTopicRequest = createTopicRequest(topicName);
+    String jsonReq = OBJECT_MAPPER.writer().writeValueAsString(clusterTopicRequest);
+    String url = "/topics/createTopics";
+    // ensure Topic has already been created in this integration test
+    executeCreateTopicRequest(jsonReq, url);
+    // attempt to create a topic that already exists.
+    MockHttpServletResponse returnedValue = executeCreateTopicRequest(jsonReq, url);
+
+    assertTrue(returnedValue.getContentAsString().contains("TopicExistsException"));
+    // verify that the response is in the correct part of the API Response otherwise it will not be
+    // found in handling by receiving clients.
+    ApiResponse apiResponse =
+        mapper.readValue(returnedValue.getContentAsString(), ApiResponse.class);
+    assertTrue(apiResponse.getResult().contains("TopicExistsException"));
+
+    embeddedKafkaBroker.doWithAdmin(
+        adminClient -> {
+          try {
+            Set<String> topicsSet = adminClient.listTopics().names().get();
+            assertThat(topicsSet).contains(topicName);
+          } catch (InterruptedException | ExecutionException e) {
+            e.printStackTrace();
```

Comment:
```suggestion
            log.error("Error : ",e);
```

Lets remove all print stack traces and replace with log.error. We have this method all over.

Reply:
Have added SLF4J to the class and changed both occurrences to this.